test/event/staking/staking-factory-events.ts

```diff
@@ -42,18 +47,18 @@ export function stakingPoolCreated(
 
     const args = event.args
 
-    expect(args?.stakingPool).is.not.undefined
-
-    expect(args?.treasury).is.not.undefined
-
     expect(args?.creator).is.not.undefined
 
-    expect(args?.rewardTokens).is.not.undefined
-    expect(args?.stakeToken).is.not.undefined
-    expect(args?.epochStartTimestamp).is.not.undefined
-    expect(args?.epochDuration).is.not.undefined
-    expect(args?.minimumContribution).is.not.undefined
-    expect(args?.rewardType).is.not.undefined
+    /* eslint-disable @typescript-eslint/no-unsafe-member-access */
```

Comment:
question: is this comment needed?

_I'm curious why the `treasury` field is an unsafe member_

Reply:
I think its because its nested inside the args.config and the linter isnt smart enough to pick up on the optional chaining

Reply:
Right, yes, I believe that may be one reason I assign to a local var to avoid the nesting 🤔 

...but if that is the cause, why are the other lines not also being flagged by the linter?
_They are also nested inside `args.config`_

Reply:
updated to assign to a local variable